packages/graphile-build-pg/src/plugins/PgBackwardRelationPlugin.ts

```diff
@@ -131,27 +132,49 @@ export default (function PgBackwardRelationPlugin(
           const isUnique = !!table.constraints.find(
             c =>
               (c.type === "p" || c.type === "u") &&
-              c.keyAttributeNums.length === keys.length &&
+              c.keyAttributeNums.length <= keys.length &&
               c.keyAttributeNums.every((n, i) => keys[i].num === n)
           );
-
           const isDeprecated = isUnique && legacyRelationMode === DEPRECATED;
 
+          const primaryKeyConstraint = table.primaryKeyConstraint;
+          const primaryKeys =
+            primaryKeyConstraint && primaryKeyConstraint.keyAttributes;
+          const uncoveredPrimaryKeys =
+            primaryKeys && keys.every(attr => primaryKeys.includes(attr))
+              ? primaryKeys.filter(attr => !keys.includes(attr))
+              : [];
+          const parameterKeys = uncoveredPrimaryKeys.map(key => ({
+            ...key,
```

Comment:
There's less risk of conflicts here if we don't splat this out:

```suggestion
            key,
```